pkg/splunk/enterprise/monitoringconsole.go

```diff
@@ -113,21 +101,26 @@ func getMonitoringConsoleDeployment(cr splcommon.MetaObject, spec *enterprisev1.
 		labels[k] = v
 	}
 
-	//create deployment configuration
-	deploymentMC := &appsv1.Deployment{
+	//create statefulset configuration
```

Comment:
As discussed, we will take up making use of GetSplunkStatefulSet and tweaking only changes in a future PR :)